intera_examples/scripts/go_to_joint_angles_in_contact.py

```diff
@@ -146,6 +146,13 @@ def main():
         "-kn", "--K_nullspace", type=float,
         nargs='+', default=[5.0, 10.0, 5.0, 10.0, 5.0, 10.0, 5.0],
         help="A list of desired nullspace stiffnesses, one for each of the 7 joints (a single value can be provided to apply the same value to all the directions) -- units are in (Nm/rad)")
+    parser.add_argument(
+        "-ddifc",  "--disable_damping_in_force_control", action='store_true', default=False,
+        help="Disable damping in force control")
+    parser.add_argument(
+        "-drr",  "--disable_reference_resetting", action='store_true', default=False,
```

Comment:
:+1: on using the store_true action and default value. I think it makes it a lot cleaner and easier to understand, and it matches the rest of the SDK.